### DISCUSSION TITLE
Build on Redhat 8 --offline

### DISCUSSION THREAD

**Discussion by smprather** [neovide/neovide#1880] [Q&A]
Is it possible to get, or to build, a neovide that will run on Redhat 8 behind firewall? Thanks!

```
$ cargo build --offline
error: failed to get `glutin` as a dependency of package `neovide v0.10.4 (/home/epramyl/neovide-0.10.4)`

Caused by:
  failed to load source for dependency `glutin`

Caused by:
  Unable to update https://github.com/neovide/glutin?branch=new-keyboard-all#7c313d91

Caused by:
  can't checkout from 'https://github.com/neovide/glutin': you are in the offline mode (--offline)
```
```
$ ./neovide.AppImage
./neovide.AppImage: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by ./neovide.AppImage)
./neovide.AppImage: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by ./neovide.AppImage)
./neovide.AppImage: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by ./neovide.AppImage)
./neovide.AppImage: /lib64/libc.so.6: version `GLIBC_2.29' not found (required by ./neovide.AppImage)
./neovide.AppImage: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by ./neovide.AppImage)
```

**Comment by fredizzimo** [comment]
Sorry for the very late reply. But I think that AppImage for the 0.11.x releasees should work, we made some changes to make it compatible with more distributions, but I don't think I ever tested Redhat 8, so there might still be problems with that.

**Comment by smprather** [comment]
Ah! I forgot that I had even written this 2 years ago. You're memory must have been jogged by my accidental dupe 😊.

Nope, it still doesn't work. Let me double check and I'll reply back with the actual output. It's just GLIBC stuff again though.

I have a new job now, and my Redhat8.10 VM can see the internet (not true in last job, --offline). But even with internet visibility, I cannot get it built. Would it be possible for you to use the FreeType interface crate instead of relying on the linker to link to external libs? I don't mind linking, I just can't get it to work.
